superset-frontend/src/components/FilterableTable/FilterableTable.tsx

```diff
@@ -95,6 +100,7 @@ const StyledFilterableTable = styled.div`
 
 // when more than MAX_COLUMNS_FOR_TABLE are returned, switch from table to grid view
 export const MAX_COLUMNS_FOR_TABLE = 50;
+export const MAX_COLUMN_WIDTH = 200;
```

Comment:
@evans was this decided somewhere as a breakpoint?
If not, @prosdev0107 consider using [this](https://stackoverflow.com/questions/61124992/is-it-possible-to-show-an-ant-design-tooltip-conditionally-only-when-the-content) if possible, which is a dynamic way of doing it.

Reply:
@diegomedina248 Evan's correct handle is @rusackas

Reply:
@michael-s-molina crap 🤦 , thanks

Reply:
THANK YOU. I spent a few minutes trying to find an example of the [AntD-native approach](https://stackoverflow.com/questions/61124992/is-it-possible-to-show-an-ant-design-tooltip-conditionally-only-when-the-content). I swear I had a CodePen example of this that I once wrote up, but I seem to have left it in my other coat. 

The 200px is indeed arbitrary. It also doesn't seem to _matter_, thus my comment/screenshot above.

I'm OK with setting a reasonable maximum column width. Exactly _how_ wide is up for grabs. Maybe @kasiazjc has opinions on the matter. We should also test/consider how the _contents_ of the cell are wrapped/truncated when hitting this maximum width.

Reply:
The only downside to the AntD approach is a little more vendor lock-in, should we ever decide to change component libraries again... but it probably won't be our biggest battle then :)

The `getTextDimension` approach seems like it'd work, but is slightly fragile-looking to me as a technique to start reusing in more places.

Reply:
Sorry for the late response, but re: columns setting max/min width can be tricky I think. To do it well we would have to go through different types of data there are in the columns and define min/max... If we would introduce max width for columns and let's say we would have 2 columns it could look weird as the table would not be stretched from left or right. 

Easy solution would be to scale the column width, so depending on the width of the table all columns always have the same width 🤔 but still, lot's of edge case.

That is, if I understand the problem correctly 🙃